conversion.py

```diff
@@ -1,2 +1,5 @@
 def celsius_to_fahrenheit(celsius):
     return celsius/5*9 + 32
+
+def celsius_to_kelvin(celcius):
+    return celsius + 273.15
```

Comment:
```suggestion
    return celsius + 273.15
    
```
Great! Suggestion to add empty line at EOF.